src/core/Akka/Event/LoggingBus.cs

```diff
@@ -85,7 +85,7 @@ protected override Type GetClassifier(object @event)
         /// <exception cref="LoggerInitializationException">
         /// This exception is thrown if the logger doesn't respond with a <see cref="LoggerInitialized"/> message when initialized.
         /// </exception>
-        internal void StartDefaultLoggers(ActorSystemImpl system)
+        internal async void StartDefaultLoggers(ActorSystemImpl system)
```

Comment:
Had to use an `async void` here to stop `Task` contamination from reaching the top of the API. Don't need `Task` error handling here either.

Open to suggestions on how to do this better if needed.